contracts/Pausable.sol

```diff
@@ -1,30 +1,31 @@
 pragma solidity >=0.4.21 <0.6.0;
 
-contract Pausable {
-  address public owner;
-  bool public paused;
+import "./Ownable.sol";
 
-  constructor() public {
-    owner = msg.sender;
-  }
+contract Pausable is Ownable {
+  event LogPaused(address account);
+  event LogUnpaused(address account);
 
-  modifier onlyOwner() {
-    require(msg.sender == owner, "Only owner can do this!");
-    _;
-  }
+  bool private paused = false;
```

Comment:
If I set `paused = true;` in the constructor, which assignment wins?

Reply:
The one in the constructor wins, in that case, it will be `true`

Reply:
More broadly, my point is that it is not good to have to do these mental gymnastics.
To be clear with the reader, and reduce the scope for errors, you should have all initialisation happen in the constructor and the constructor only.

Reply:
Fixed by 5296ae8b788461bb2e976000dc9cbfd3132a8708

Reply:
Fixed by 5296ae8b788461bb2e976000dc9cbfd3132a8708